app/labor/hex_comparer.rb

```diff
@@ -1,49 +1,49 @@
 class HexComparer
+  RGB_REGEX = /^#?(?<r>..)(?<g>..)(?<b>..)$/.freeze
+  ACCENT_MODIFIERS = [1.14, 1.08, 1.06, 0.96, 0.9, 0.8, 0.7, 0.6].freeze
+
   def initialize(hexes, amount = 1)
-    @hexes = hexes
+    @hexes = hexes.sort
     @amount = amount
   end
 
-  def order
-    hexes.sort
-  end
-
   def smallest
-    order.first
+    hexes.first
   end
 
   def biggest
-    order.last
+    hexes.last
   end
 
   def brightness(amount = 1)
-    rgb = smallest.delete("#").scan(/../).map(&:hex).map { |color| color * amount }.map(&:round)
-    format("#%<r>02x%<g>02x%<b>02x", r: rgb[0], g: rgb[1], b: rgb[2])
+    rgb = hex_to_rgb_hash(smallest).transform_values do |color|
```

Comment:
Leaning a bit more on Ruby here. `scan` followed by 3 map operations wasn't particularly efficient, this seems easier and I can reuse `hex_to_rgb_hash` in `opacity`.